neural_compressor/tensorflow/quantization/tf_utils/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024 Intel Corporation
```

Comment:
As it is already under `tensorflow` folder, please remove `tf_` or `tensorflow_` prefix.